### PR TITLE
New Kokoro tests including Trampoline

### PR DESCRIPTION
**Pull request by gogasca**
Add new tests for Census using Trampoline
@kurtisvg GiHub PR

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/422)

### CODE REVIEW

File: .kokoro/tests/census.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Copyright 2019 Google Inc.
```
Comment (kurtisvg) [inline .kokoro/tests/census.sh]:
```suggestion
# Copyright 2019 Google LLC
```

File: .kokoro/trampoline.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2019 Google Inc.
```
Comment (kurtisvg) [inline .kokoro/trampoline.sh]:
```suggestion
# Copyright 2019 Google LLC
```

File: .kokoro/census/common.cfg [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+# Use the trampoline script to run in docker.
+build_file: "cloudml-samples/.kokoro/trampoline.sh"
+
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/python"
+}
+
+env_vars: {
```
[the remark this answers is not in the document]
Comment (gogasca) [reply .kokoro/census/common.cfg]: Thanks for the review, I updated the original in internal CL.

File: .kokoro/census/common.cfg [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    key: "GCP_PROJECT"
+    value: "cloudml-samples"
+}
+
+env_vars: {
+    key: "CMLE_REPO_DIR"
+    value: "${KOKORO_ARTIFACTS_DIR}/github/cloudml-samples"
```
[the remark this answers is not in the document]
Comment (gogasca) [reply .kokoro/census/common.cfg]: Updated new env variable.

[the remark this answers is not in the document]
Comment (gogasca) [reply .kokoro/census/common.cfg]: I updated common.cfg with used variables. Now that presubmit.cfg, periodic.cfg and continuous.cfg are empty doe sit makes sense to delete them?

**Comment by gogasca** [comment]
@kurtisvg thanks for the review, made changes suggested, PTAL. Thanks

File: .kokoro/census/common.cfg (hunk 4 above)
[the remark this answers is not in the document]
Comment (kurtisvg) [reply .kokoro/census/common.cfg]: You'll want to leave them, even if empty. The builds won't start without them.

**Comment by gogasca** [comment]
Thanks @kurtisvg

**Pull request merged**